src/modules/dashboard/components/ColonyHome/ColonyExtensions/ColonyExtensions.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import Heading from '~core/Heading';
+import { Colony, useColonyExtensionsQuery } from '~data/index';
+import { SpinnerLoader } from '~core/Preloaders';
+import extensionData from '~data/staticData/extensionData';
+
+import styles from './ColonyExtensions.css';
+import NavLink from '~core/NavLink';
+import ExtensionStatus from '~dashboard/Extensions/ExtensionStatus';
+
+const MSG = defineMessages({
+  title: {
+    id: 'dashboard.ColonyHome.ColonyExtensions.title',
+    defaultMessage: 'Enabled extensions',
+  },
+});
+
+interface Props {
+  colony: Colony;
+}
+
+const displayName = 'dashboard.ColonyHome.ColonyExtensions';
+
+const ColonyFunding = ({ colony: { colonyName, colonyAddress } }: Props) => {
```

Comment:
This should be called ColonyExtensions, right?

Reply:
Good catch